### PR TITLE
Move math

### PR DESCRIPTION
**Pull request by Kayvv**
Move scaffoldmaker vector functions to vectorops.py

### CODE REVIEW

**Review by rchristie — CHANGES_REQUESTED** [review]
A first change is to increment version of cmlibs.maths to 0.6.0 and make that a dependency in scaffoldmaker setup.py.

File: src/scaffoldmaker/meshtypes/meshtype_3d_ostium2.py [hunk 1]
```diff
@@ -560,13 +560,13 @@ def generateOstiumMesh(region, options, trackSurface, centralPath, startNodeIden
...
-    #         nd1.insert(1, vector.setMagnitude(md1, elementLengthAroundOstiumMid if (0 < iv < (vesselsCount - 2)) else
+    #         nd1.insert(1, set_magnitude(md1, elementLengthAroundOstiumMid if (0 < iv < (vesselsCount - 2)) else
     #                                                elementLengthAroundOstiumEnd))
-    #         nd2.insert(1, vector.setMagnitude(md2, ostiumRadius))
+    #         nd2.insert(1, set_magnitude(md2, ostiumRadius))
     #     nd2 = interp.smoothCubicHermiteDerivativesLine(nx, nd2, fixAllDirections=True)
     #     px, pd2, pe, pxi = interp.sampleCubicHermiteCurves(nx, nd2, elementsCountAcross)[0:4]
     #     pd1 = interp.interpolateSampleLinear(nd1, pe, pxi)
-    #     pd3 = [vector.setMagnitude(vector.crossproduct3(pd1[n2], pd2[n2]), commonOstiumWallThickness)
+    #     pd3 = [set_magnitude(vector.crossproduct3(pd1[n2], pd2[n2]), commonOstiumWallThickness)
```
Comment (hsorby) [inline src/scaffoldmaker/meshtypes/meshtype_3d_ostium2.py:569]: Missed one.

File: src/scaffoldmaker/meshtypes/meshtype_3d_ostium2.py [hunk 2]
```diff
@@ -579,7 +579,7 @@ def generateOstiumMesh(region, options, trackSurface, centralPath, startNodeIden
     #         od2[n3][oa] = [-d for d in ld2[0]]
     #         od2[n3][ob] = ld2[-1]
     #         if useCubicHermiteThroughOstiumWall:
-    #             pd3Element = [vector.setMagnitude(vector.crossproduct3(pd1[n2], pd2[n2]),
+    #             pd3Element = [set_magnitude(vector.crossproduct3(pd1[n2], pd2[n2]),
```
Comment (hsorby) [inline src/scaffoldmaker/meshtypes/meshtype_3d_ostium2.py:582]: Missed one.

File: src/scaffoldmaker/meshtypes/meshtype_3d_uterus1.py [hunk 3]
```diff
@@ -1758,25 +1757,25 @@ def getDoubleTubeNodes(cx_tube_group, elementsCountAlong, elementsCountAround, e
...
     for n in range(len(cx_tube_group[0])):
         x = cx_tube_group[0][n]
-        v2r = vector.normalise(cx_tube_group[2][n])
-        v3r = vector.normalise(cx_tube_group[4][n])
-        d2Mag = vector.magnitude(cx_tube_group[2][n])
+        v2r = normalize(cx_tube_group[2][n])
+        v3r = normalize(cx_tube_group[4][n])
+        d2Mag = magnitude(cx_tube_group[2][n])
         r2 = (d2Mag - distance - wallThickness) / 2
         # d3Mag = vector.magnitude(cx_tube_group[4][n])
```
Comment (hsorby) [inline src/scaffoldmaker/meshtypes/meshtype_3d_uterus1.py:1764]: Should change this as well.

**Review by hsorby — CHANGES_REQUESTED** [review]

**Pull request closed without merging**